docs/docs/migrations.md

```diff
@@ -306,6 +306,74 @@ queryInterface.removeIndex('Person', 'SuperDuperIndex')
 queryInterface.removeIndex('Person', ['firstname', 'lastname'])
 ```
 
+### addConstraint(tableName, indexNameOrAttributes, options)
```

Comment:
Sorry. It should just be `attributes`. The signature in the code is correct. But i missed it here. Will change it